### ISSUE TITLE
Option to turn off or change the Stories background/shadow color 

### ISSUE THREAD

**Issue by LuckynaSan** [ampproject/amphtml#35638]
### Description

As a user, I want the ability to turn on/off the story's shadow background color, or the ability to set the color on each page's shadow.

![image](https://user-images.githubusercontent.com/66372350/129214277-b5871286-e94c-4a52-a373-83ee49fbad67.png)

### Alternatives Considered

N/A

### Additional Context

This was requested by a Web Stories user: https://wordpress.org/support/topic/colored-blurred-background-on-web-browser-version/

**Comment by processprocess** [comment]
Thanks for filing this @LuckynaSan and reporting this from the WP support page.

We've been discussing this feature and this user feedback is very helpful for thinking about next steps. 
I'll share this with the team and will post any updates here.

cc @hongcatlover

**Comment by ArashA007** [comment]
YES, would be great to get rid of the blurred background. The blurred background should be a tick on/off.
